src/handler/structures/errors.ts

```diff
@@ -1,9 +1,34 @@
+/**
+ * @enum { string }
+ */
 export enum SernError {
+    /**
+     * Throws when registering an invalid module.
+     * This means it is undefined or an invalid command type was provided
+     */
     NonValidModuleType = 'Detected an unknown module type',
```

Comment:
`NonValidModuleType` or `InvalidModuleType`
what do you think

Reply:
InvalidModuleType is better

Reply:
This would be a breaking change, correct?

Reply:
does it break code of current cjs users?

Reply:
i assume you'll rename and commit, resolving this @jacoobes

Reply:
yes